mage/testdata/compiled/custom.go

```diff
@@ -14,7 +14,9 @@ var Default = Deploy
 
 // This is very verbose.
 func TestVerbose() {
-	log.Println("hi!")
+	if mg.Verbose() {
+		log.Println("hi!")
+	}
```

Comment:
log's output is already determined by the flag, so I think this needs to be a separate test, so that we're not munging the two tests.  One should test that log does the right thing and one should test that mg.Verbose does the right thing.

Reply:
Done